In [1]:
import numpy as np # linear algebra 
import pandas as pd # data processing, CSV file I/O (e.g. pd.read_csv)
# stats models: regression fitting via formulas
import statsmodels.formula.api as smf
# stats models: regression fitting via matrices of regression design
import statsmodels.api as sm

In [2]:
df = pd.read_csv('https://raw.githubusercontent.com/artamonoff/Econometrica/master/python-notebooks/data-csv/Labour.csv')
df.head()

capital  labour     output       wage
0   2.606563     184   9.250759  43.080307
1   1.323237      91   3.664310  27.780016
2  22.093692     426  28.781516  44.467748
3  10.737851      72   4.124642  39.734710
4   1.161365      46   2.890150  34.650709

In [4]:
# специфицируем модель через формулу
output_eq1 = smf.ols(formula='np.log(output)~np.log(capital)+np.log(labour)+np.log(wage)', data=df).fit()
# Коэфициенты модели с округление
output_eq1.params.round(3)

Intercept         -5.007
np.log(capital)    0.149
np.log(labour)     0.720
np.log(wage)       0.921
dtype: float64

### Спецификация 

$$
np.log(output)= -5.007+0.149*np.log(capital)+0.720*np.log(labour)+0.921*np.log(wage)
$$

### Интерпретация
1. При увеличении капитала компании на 1%, выпуск продукции увелоичится на 0.149% при прочих равных условиях.
2. При увеличении количества сотрудников на 1%, выпуск продукции увеличится на 0.72% при прочих равных условиях. 